scripts/cat/skills.py

```diff
@@ -366,17 +366,17 @@ def generate_new_catskills(status, moons, hidden_skill: HiddenSkillEnum = None):
         new_skill.hidden = hidden_skill
 
         # TODO: Make this nicer
-        if status == "newborn" or moons <= 0:
+        if status.is_newborn() or moons <= 0:
             pass
-        elif status == "kitten" or moons < 6:
+        elif status.is_kitten() or moons < 6:
             new_skill.primary = Skill.get_random_skill(points=0, interest_only=True)
-        elif status == "apprentice":
+        elif status.is_warrior_app():
             new_skill.primary = Skill.get_random_skill(point_tier=1, interest_only=True)
             if random.randint(1, 3) == 1:
                 new_skill.secondary = Skill.get_random_skill(
                     point_tier=1, interest_only=True, exclude=new_skill.primary.path
                 )
-        elif moons < 50:
+        elif moons < 50:  # Should these thresholds be in a config file?
```

Comment:
Open a new issue or ask any of us Senior Developers in the Discord; comments like this sadly tend to remain in the code forever untouched.

Reply:
(Or turn it into a TODO comment with a clear description of the task like you did below)

Reply:
TODO-ified